File: P86491.py
def solution(sizes):
    answer = 0
    w, h = 0, 0
    for size in sizes:
        w_, h_ = size[0], size[1]
        if h_ > w_:
            w_, h_ = h_, w_
        if w_ > w:
            w = w_
        if h_ > h:
            h = h_
    answer = w*h
    return answer

File: P86491.java
public class P86491 {
    public int solution(int[][] sizes) {
        int answer = 0;
        int w = 0;
        int h = 0;
        for (int[] size : sizes) {
            int W = size[0];
            int H = size[1];
            
            if (H > W) {
                int tmp = H;
                H = W;
                W = tmp;
            }
            
            if (W > w) w = W;
            if (H > h) h = H;
        }
        
        answer = w*h;
        return answer;
    }
}
